MAGECLOUD-2899: Redis Slave Nodes Have no Health Checks / Are not Stable (#4228)
magento/devdocs: guides/v2.1/cloud/env/variables-deploy.md

You must have a Redis service configured in the `.magento.app.yaml` file and in the `services.yaml` file.

> [ece-tools version 2002.0.18]({{ page.baseurl }}/cloud/release-notes/cloud-tools.html#v2002018) and more , uses more fault tolerant settings. In the case when the Magento 2 cannot read data from the Redis slave instance, the reading will be made from the Redis master instance.

Reviewer: ```suggestion
[ece-tools version 2002.0.18]({{ page.baseurl }}/cloud/release-notes/cloud-tools.html#v2002018) uses more fault-tolerant settings. If Magento 2 cannot read data from the Redis _slave_ instance, then it reads data from the Redis _master_ instance.
```
